packages/assets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@atb-as/generate-assets",
-  "version": "10.1.0",
+  "version": "10.1.1-alpha.0",
```

Comment:
We might want to bump the version to something none-alpha too?

Reply:
I'll do this when doing a proper release. This is handled by lerna for publishing new versions (did a release from the branch for easier testing in your case)